projects/packages/changelogger/changelog/fix-bump_composer_versions_round2

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: changed
+
+Updated dependencies.
```

Comment:
More important to mention, IMO:
```suggestion
Significance: patch
Type: changed

Bump minimum PHP version from implicitly-7.2.0 to 7.2.5.
```
Technically it should possibly be major since it's dropping some PHP version support, but we just dropped 7.0 and 7.1 and this could be considered a bug in that we weren't explicit enough then. 🤷

And then there are those who think dropping php versions doesn't matter at all since Composer will just not install the later version anyway. 🤷